util/package.js

```diff
@@ -1,5 +1,32 @@
 'use strict';
 
+var allStars = require( 'all-stars' );
+
+/**
+ * Check all-stars db for given person and normalize person data if found.
+ *
+ * @param  {Object|false} person object representing author or maintainer
+ *
+ * @return {Object|false}        same object given, possibly modified or augmented
+ */
+function getAllStar( person ) {
+  if ( !person ) return person;
+
+  var allStar = allStars( person );
+  if ( allStar ) {
```

Comment:
I will change this, but keep in mind that it will couple the object structure returned by `credits` to the object structure returned by `all-stars`. (And you may get more info than you want.) :smile:

Here's an example:

```
{ name: 'Benjamin Coe',
  email: 'ben@npmjs.com',
  id: 'bcoe',
  npmUsers: [ 'bcoe' ],
  npmUser: 'bcoe',
  emails: [ 'ben@npmjs.com', 'bencoe@gmail.com' ],
  names: [ 'Benjamin Coe', 'Ben Coe', 'Benjamin E. Coe' ],
  githubUsers: [ 'bcoe' ],
  githubUser: 'bcoe',
  twitters: [ 'benjamincoe' ],
  twitter: 'benjamincoe',
  summary: [Function: summary],
  toString: [Function: toString],
  packages: 
   [ 'cliui',
     'os-locale',
     'y18n',
     'signal-exit',
     'nyc',
     'yargs',
     'wrap-ansi' ] }
```